bonsai/Bonsai.config

```diff
@@ -29,7 +29,7 @@
     <Package id="Bonsai.Shaders" version="0.26.0" />
     <Package id="Bonsai.Shaders.Design" version="0.26.0" />
     <Package id="Bonsai.Shaders.Rendering" version="0.3.1" />
-    <Package id="Bonsai.Spinnaker" version="0.7.0" />
+    <Package id="Bonsai.Spinnaker" version="0.7.1" />
```

Comment:
Why update the bonsai package?

Reply:
I was having trouble running on my computer. This can probably be skipped.